examples/sveltekit-ts/src/lib/Counter.svelte

```diff
@@ -1,20 +1,22 @@
 <script lang="ts">
 	import { spring } from 'svelte/motion';
 
-	let count = 0;
+	let count = $state(0);
 
 	const displayed_count = spring();
-	$: displayed_count.set(count);
-	$: offset = modulo($displayed_count, 1);
 
 	function modulo(n: number, m: number) {
 		// handle negative numbers
 		return ((n % m) + m) % m;
 	}
+	$effect(() => {
+		displayed_count.set(count);
```

Comment:
You might try generating a new project and copying these files over. I notice the capitalization is different here than it would be in a new project where `displayedCount` would be used. Potentially there are other differences as well